packages/types/src/attendance.ts

```diff
@@ -99,6 +99,49 @@ export const AttendanceWriteSchema = AttendanceSchema.pick({
   selections: true,
 })
 
+export type DeregisterReason = z.infer<typeof DeregisterReasonSchema>
+export const DeregisterReasonSchema = schemas.DeregisterReasonSchema
+
+export type DeregisterReasonWrite = z.infer<typeof DeregisterReasonWriteSchema>
+export const DeregisterReasonWriteSchema = DeregisterReasonSchema.pick({
+  type: true,
+  details: true,
+  userId: true,
+  eventId: true,
+  registeredAt: true,
+  userGrade: true,
+})
+
+export type DeregisterReasonFormWrite = z.infer<typeof DeregisterReasonFormWriteSchema>
+export const DeregisterReasonFormWriteSchema = DeregisterReasonWriteSchema.pick({
+  type: true,
+  details: true,
+})
```

Comment:
we havent shared form types from server/client before. maybe just duplicate the types for now?